### PR TITLE
Update the maven group and document how to add the library as a dependency

### PR DESCRIPTION
**Pull request by jkwatson**
Until we are publishing to maven local, this documents work-arounds for consuming the library.

### CODE REVIEW

**Review by breedx-splk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -35,6 +35,74 @@
...
+allprojects {
+    repositories {
+        google()
+...
+        mavenLocal()
+    }
+}
+```
+
+Then, add the locally built library as a dependency in your application's build.gradle:
```
Comment (breedx-splk) [inline README.md:96]: `build.gradle.kts` if android uses kotlin for building too eh?

Comment (jkwatson) [reply README.md:96]: no one uses that in real life.  ;)

Comment (jkwatson) [reply README.md:96]: on a serious note, though...the people who know how to write build.gradle.kts will be able to sort out the conversion. :)

Comment (jkwatson) [reply README.md:96]: also, this is hopefully super temporary until we get maven central publishing working.

Comment (jkwatson) [reply README.md:96]: Just as another data point, Android Studio generates build.gradle for new projects, so I'm mostly targetting that most-common configuration.

**Pull request merged**